add description of filter tags env variables (#10006)
DataDog/documentation: content/en/agent/docker/apm.md

| `DD_PROXY_HTTPS`           | Set up the URL for the proxy to use.                                                                                                                                                                                                                                                                                                                 |
| `DD_APM_REPLACE_TAGS`      | [Scrub sensitive data from your span’s tags][2].                                                                                                                                                                                                                                                                                                     |
| `DD_APM_FILTER_TAGS_REQUIRE`      | Defines required tags that traces must have in order to be sent to Datadog.                                                                                                                                                                                                                                                                                                     |

> | `DD_APM_FILTER_TAGS_REJECT`      | Defines rejected tags that traces must not have and are dropped by the Agent.                                                                                                                                                                                                                                                                                                     |

Reviewer: ```suggestion
| `DD_APM_FILTER_TAGS_REJECT`      | Defines rejection tags. The Agent drops traces that have these tags.       |
```
